quill-sql/src/main/scala/io/getquill/context/QueryExecution.scala

```diff
@@ -111,7 +111,7 @@ object Execution:
       case None =>
         GenericDecoder.summon[DecoderT, ResultRow, Session]
 
-  /** See if there there is a QueryMeta mapping T to some other type RawT */
+  /** See if there is a QueryMeta mapping T to some other type RawT */
```

Comment:
Correcting doubled words is slightly more challenging than word corrections. I'm pretty sure this is ok.